docs/mfc/reference/cchecklistbox-class.md

```diff
@@ -44,7 +44,21 @@ A "checklist box" displays a list of items, such as filenames. Each item in the
 
 `CCheckListBox` is only for owner-drawn controls because the list contains more than text strings. At its simplest, a checklist box contains text strings and check boxes, but you don't need to have text at all. For example, you could have a list of small bitmaps with a check box next to each item.
 
-To create your own checklist box, you must derive your own class from `CCheckListBox`. To derive your own class, write a constructor for the derived class, then call `Create`.
+The basic steps to create your own checklist box are as follows:
+
+First design an ordinary listbox control using the resource editor (because that's what the checklist box is based upon).
+The listbox must have 'Owner Draw' set to `Fixed` and `Has Strings` to `True`.
+That's because the checklist box needs to draw the checkboxes.
```

Comment:
These three sentences will be rendered "back-to-back" (i.e. without the newline seen in the raw markdown). Therefore, these sentences should be collapsed into the one line, such that the author's intent on the format matches the rendered output. This applies to every such occurrence in this PR. Thanks.